model/Licensing/Classes/SimpleCustomLicense.md

```diff
@@ -0,0 +1,26 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# SimpleCustomLicense
```

Comment:
`SimpleLicensingText`

Reply:
I took the liberty to also change the corresponding property name, I hope that was in your interest, too.